recipes/super_distance/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version="0.1" %}
+
+
+package:
+  name: super_distance 
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/quadram-institute-bioscience/super_distance.git
+  md5: unused
+
+build:
+  number: 0
+  skip: true # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - automake
+    - make
+    - check
+    - libcxx 4.0.* # [osx]
+    - openmp 4.0.* # [osx]
+    - llvm-openmp # [osx]
```

Comment:
Can you remove the 3 OSX lines? Since OSX is being skipped they're not needed.

Reply:
You can remove them from both the build and host sections :)

Reply:
done! tks for checking